spec/Section 7 -- Response.md

```diff
@@ -135,14 +145,103 @@ locations, where each location is a map with the keys `line` and `column`, both
 positive numbers starting from `1` which describe the beginning of an
 associated syntax element.
 
+If an error can be associated to a particular field in the GraphQL result, it
+should contain an entry with the key `path` that details the path of the
```

Comment:
Should we should tighten up this language from should to must? Ideally the `path` is always provided in relevant error cases by all implementations. Should gives an opt-out, but I'm not sure if there's a reasonable opt-out rationale here.

If you agree, perhaps this should be:

```
If an error is associated with executing a particular field in a GraphQL 
result (referred to elsewhere as a field error), it must contain an entry 
with the key `path` that details the path of the
```